posts/2022-12-06-22.0.0.13-beta.adoc

```diff
@@ -0,0 +1,353 @@
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Jakarta EE 10 Platform and MicroProfile 6
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+With Open Liberty 22.0.0.13-beta we're excited to announce many new features and functions, beginning with Jakarta EE Web Profile 10.0 and Jakarta EE Platform 10.0! To make that possible, this release also includes the last remaining Jakarta EE 10 features that had not previously been part of a beta: Jakarta Authentication 3.0, Jakarta Authorization 2.1, Jakarta Security 3.0, and Jakarta Faces 4.0.
+
+
+Similar to Jakarta EE 10 features, we've also been releasing various MicroProfile 6 features in the recent beta releases, and the 22.0.0.13-beta marks the first release that includes all of MicroProfile 6!  Also provided in this beta release is the ability to automatically purges FFDC log files after they reach a configured age.
+
+
+// // // // // // // //
+// Change the RELEASE_SUMMARY to an introductory paragraph. This sentence is really
+// important because it is supposed to grab the readers attention.  Make sure to keep the blank lines 
+//
+// Throughout the doc, replace 22.0.0.13-beta with the version number of Open Liberty, eg: 22.0.0.2-beta
+// // // // // // // //
+
+The link:{url-about}[Open Liberty] 22.0.0.13-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+* <<jakarta10, Jakarta EE 10 Platform and Web Profile>> 
+** <<authentication, Application Authentication 3.0 (Jakarta Authentication 3.0)>>
+** <<authorization, Application Authorization 2.1 (Jakarta Authorization 2.1)>>
+** <<security, Application Security 5.0 (Jakarta Security 3.0)>>
+** <<faces, Jakarta Faces 4.0>>
+* <<mp6, MicroProfile 6.0>>
+* <<maxFfdcAge, Configurable maximum FFDC age>>
+
+
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/23358
+// Contact/Reviewer: ReeceNana,jhanders34
+// // // // // // // // 
+[#jakarta10]
+== Jakarta EE 10 Platform and Web Profile
+
+In the Open Liberty 22.0.0.9-beta, we made available Jakarta EE 10 Core Profile and ever since we've been hard at work releasing more Jakarta EE 10 features with each subsequent beta.  We're proud to announce that the 22.0.0.13-beta is the culmination of that effort and makes available Jakarta EE 10.0 Application Client, Jakarta EE Web Profile 10.0, and Jakarta EE Platform 10.0.

```

Comment:
```suggestion
We released the Jakarta EE 10 Core Profile in the Open Liberty 22.0.0.9-beta and ever since we've been hard at work releasing more Jakarta EE 10 features with each subsequent beta.  We're proud to announce that the 22.0.0.13-beta is the culmination of that effort and makes available Jakarta EE 10.0 Application Client, Jakarta EE Web Profile 10.0, and Jakarta EE Platform 10.0.

```